main.go

```diff
@@ -85,7 +85,7 @@ func main() {
 	}
 
 	// initialize libhoney
-	libhoney.UserAgentAddition = fmt.Sprintf("honeycomb-lambda-extension/%s", version)
+	libhoney.UserAgentAddition = fmt.Sprintf("honeycomb-lambda-extension-<arch>/%s", version)
```

Comment:
![yes](https://user-images.githubusercontent.com/517302/135665637-bbed6d6c-c5a1-40a4-a44b-d03657fde88b.gif)

Reply:
Oh oh. Fan of the Parens, here. Would this work?

```suggestion
	libhoney.UserAgentAddition = fmt.Sprintf("honeycomb-lambda-extension/%s (<arch>)", version)
```